catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -185,6 +186,20 @@ public UpdateResponse version(UpdateResponse updateResponse) {
           response.getCreatedMetacards().stream().map(Metacard::getId).collect(TO_A_STRING));
     }
 
+    if (response.getCreatedMetacards().size() == updateResponse.getUpdatedMetacards().size()) {
+      for (int i = 0; i < updateResponse.getUpdatedMetacards().size(); i++) {
+        Attribute versionIdAttribute = response.getCreatedMetacards().get(i).getAttribute("id");
+        updateResponse
+            .getUpdatedMetacards()
+            .get(i)
```

Comment:
`updateResponse.getUpdatedMetacards` could be called once above the for loop right? Then you could loop through them?

Reply:
same for `response.getCreatedMetacards()`?